### PR TITLE
Add notes about rendering ads on Windows a11y laptops

### PR DESCRIPTION
**Pull request by DenisHdz**
Resolves n/a

**Overall change:**
During the [Canonical Ads swarm](https://github.com/bbc/simorgh/issues/5956), we were unable to set a proxy on the Windows a11y laptop, so we had to find a workaround.

This PR adds a new section to the README explaining how to do this, given the case.

**Code changes:**
- Expand the README with a new `Rendering Ads on an a11y device` section

---

- [X] I have assigned myself to this PR and the corresponding issues
- [X] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [X] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by itimirichard** [comment]
Thanks for this Denis. My understanding from this doc is that simorgh will be served from one machine and accessed via the windows a11y laptop. My question is do both devices need to be on the same connection for this to work?

**Comment by DenisHdz** [comment]
> Thanks for this Denis. My understanding from this doc is that simorgh will be served from one machine and accessed via the windows a11y laptop. My question is do both devices need to be on the same connection for this to work?

Yes, they do! I'll add that as well.

File: docs/Rendering-Ads.md [hunk 1]
```diff
@@ -31,12 +33,32 @@ Given you are using Chrome browser.
...
-1. Use a proxy to set your location outside the UK, visiting https://www.test.bbc.com/userinfo should show `"X-Ip_is_advertise_combined": "yes"`. And when you visit https://www.test.bbc.com/mundo you should be able to see an advert.
+1. Use a proxy to set your location outside the UK. Visiting https://www.test.bbc.com/userinfo should show `"X-Ip_is_advertise_combined": "yes"`. And when you visit https://www.test.bbc.com/mundo you should be able to see an advert.
 
 2. If you would like to see the ads with the debugger tool you can append this query string to the url: `?ads-debug=true`
 
    i.e https://www.test.bbc.com/mundo?ads-debug=true
 
+## Rendering Ads on an a11y device
+
+If you are unable to set a proxy on one of the Windows a11y laptops, you won't be able to see ads within the UK. For this reason, you'll have to run Simorgh locally in your working machine and access to it from the Windows laptop following these steps:
```
Comment (hotinglok) [inline docs/Rendering-Ads.md]:
```suggestion
If you are unable to set a proxy on one of the Windows a11y laptops, you won't be able to see ads within the UK. For this reason, you'll have to run Simorgh locally on your working machine (Mac) and allow the Windows laptop to connect to it over LAN with the following steps:
```

File: docs/Rendering-Ads.md [hunk 2]
```diff
@@ -31,12 +33,32 @@ Given you are using Chrome browser.
...
+
+1. Connect both devices to the same network
+
+2. Update the `SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN` url in the [local.env](https://github.com/bbc/simorgh/blob/latest/envConfig/local.env#L3) file with the IP address of your machine
```
Comment (hotinglok) [inline docs/Rendering-Ads.md:48]:
```suggestion
2. Update the `SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN` url in the [local.env](https://github.com/bbc/simorgh/blob/latest/envConfig/local.env#L3) file with the IP address of your machine (Mac)
```

File: docs/Rendering-Ads.md [hunk 3]
```diff
@@ -31,12 +33,32 @@ Given you are using Chrome browser.
...
+
+   i.e `SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN=http://192.168.0.50:7080`
```
Comment (hotinglok) [inline docs/Rendering-Ads.md]:
```suggestion
   e.g `SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN=http://192.168.0.50:7080`
```

File: docs/Rendering-Ads.md [hunk 4]
```diff
@@ -31,12 +33,32 @@ Given you are using Chrome browser.
...
+
+3. Disable the `upgrade-insecure-requests` directive from the [cspHeader](https://github.com/bbc/simorgh/blob/latest/src/server/utilities/cspHeader/index.js#L419)
+
+4. If you can't set the `BBC-Adverts` header in the Windows laptop, you will need to remove [this](https://github.com/bbc/simorgh/blob/latest/src/app/containers/Ad/Canonical/index.jsx#L63) condition from the codebase to be able to render the ad
+
+5. Run Simorgh with a production build `npm run build && npm run start`
+
+6. Access to your machine from the Windows laptop
```
Comment (hotinglok) [inline docs/Rendering-Ads.md]:
```suggestion
6. Connect to your machine running Simorgh using the Windows laptop by visiting the IP address of it appended with the correct query strings.
```

File: docs/Rendering-Ads.md [hunk 5]
```diff
@@ -31,12 +33,32 @@ Given you are using Chrome browser.
...
+
+   i.e `http://192.168.0.50:7080/mundo?site=test&ads-debug=true`
```
Comment (hotinglok) [inline docs/Rendering-Ads.md]:
```suggestion
   e.g `http://192.168.0.50:7080/mundo?site=test&ads-debug=true`
```

**Review by hotinglok — COMMENTED** [review]
Minor changes, thanks for the write up!

File: docs/Rendering-Ads.md (hunk 2 above)
Comment (DenisHdz) [reply docs/Rendering-Ads.md:48]: I'd prefer not to add Mac, since there could be people using Window machines.

**Comment by DenisHdz** [comment]
> Minor changes, thanks for the write up!

Thanks for the suggestions 👍

**Review by amywalkerdev — APPROVED** [review]

**Review by hotinglok — APPROVED** [review]
LGTM, nice write up!

**Pull request merged**